src/Uart/UartRegisters.h

```diff
@@ -71,41 +71,47 @@ typedef volatile struct UartRegisters
 
 // clang-format off
 
-#define UART_DATA_OFFSET    0x00U
-
-#define UART_STATUS_OFFSET  0x04U
-#define UART_STATUS_RCNT    0xFC000000U  // Receiver FIFO count
-#define UART_STATUS_TCNT    0x3F000000U  // Transmitter FIFO count
-#define UART_STATUS_RF      0x400U       // Receiver FIFO full
-#define UART_STATUS_TF      0x200U       // Transmitter FIFO full
-#define UART_STATUS_RH      0x100U       // Receiver FIFO half-full
-#define UART_STATUS_TH      0x80U        // Transmitter FIFO half-full
-#define UART_STATUS_FE      0x40U        // Framing error
-#define UART_STATUS_PE      0x20U        // Parity error
-#define UART_STATUS_OV      0x10U        // Overrun
-#define UART_STATUS_BR      0x8U         // Break received
-#define UART_STATUS_TE      0x4U         // Transmitter FIFO empty
-#define UART_STATUS_TS      0x2U         // Transmitter shift register empty
-#define UART_STATUS_DR      0x1U         // Data ready
-
-#define UART_CONTROL_OFFSET 0x08U
-#define UART_CONTROL_FA     0x80000000U // FIFOs available
-#define UART_CONTROL_DB     0x800U      // FIFO debug mode
-#define UART_CONTROL_RF     0x400U      // Receiver FIFO interrupt
-#define UART_CONTROL_TF     0x200U      // TransmitteR FIFO interrupt
-#define UART_CONTROL_LB     0x80U       // Loopback
-#define UART_CONTROL_PE     0x20U       // Parity
-#define UART_CONTROL_PS     0x10U       // Parity select
-#define UART_CONTROL_TI     0x8U        // Transmitter interrupt enable
-#define UART_CONTROL_RI     0x4U        // Receiver interrupt enable
-#define UART_CONTROL_TE     0x2U        // Transmitter enable
-#define UART_CONTROL_RE     0x1U        // Receiver enable
-
-#define UART_CLKSCL_OFFSET  0x0CU
-#define UART_CLKSCL_DIV     8U          // Clock divider value for baud rate generation
-#define UART_CLKSCL_SHIFT   4U          // Byte shift to convert 12-bit clkscl register value to 16-bit
-
-#define UART_FIFO_OFFSET    0x10U
+#define UART_DATA_OFFSET    0x00u
+
+#define UART_STATUS_OFFSET  0x04u
+typedef enum { UART_DR = 0,      // Data ready: Indicates that new data is available in the receiver holding register.
+               UART_TS,          // Transmitter shift register empty: Indicates that the transmitter shift register is empty.
```

Comment:
I would really love here to have UART_STATUS_X, UART_CONTROL_Y here. Or UART_SR_X, UART_CR_Y. Please see 
https://github.com/n7space/SAMV71-BSP/blob/main/src/Uart/UartRegisters.h
(I agree with the enum approach, it has its benefits, but please try to use more explicit naming)